package.json

```diff
@@ -2,21 +2,35 @@
   "name": "glslify-loader",
   "version": "1.0.2",
   "description": "glslify loader module for webpack",
-  "main": "index.js",
+  "main": "glslify-loader.js",
   "license": "MIT",
   "scripts": {
-    "start": "webpack-dev-server --colors",
```

Comment:
Why was the dev server build removed?

Reply:
[webpack-serve](https://github.com/webpack-contrib/webpack-serve) has been deprecated now...